staff/marc-sarroca/cv/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Curriculum</title>
+  </head>
+  <body>
+    <h1>Curriculum Vitae</h1>
+    <img
+      src="https://cdn11.bigcommerce.com/s-1ah1x75lfn/products/890/images/2579/rickpolo__21335.1504202837.500.750.jpg?c=2"
+      alt="Smiley face"
+      height="150"
+      width="150"
+    />
+    <address>
+      <p>Rick Penciner</p>
+      <b>Date of birth:</b> 30/2/1957 <br /><b>City:</b> Las Vegas
+    </address>
+    <a><a href="mailto:rick@pawn.com">Email</a></a>
```

Comment:
I think you could delete the first a tag.

Reply:
Do it!